### ISSUE TITLE
[Accessibility: ACC-261.6] Button: Pinned state has less CC than 3:1

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#12565]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
In default themes,  graphical objects (like charts) and visual details which identify a UI element and its state should have minimum contrast ratio of 3.0:1.
A minimum contrast improves legibility for partially sighted users (for example, users with low visual acuity or color vision deficiencies) and makes content easier to use. Therefore, all visual details (such as lines, colors, …) that identify a UI element, show its state or indicate its interoperability need to fulfill a minimum contrast.

<img width="452" alt="image" src="https://github.com/user-attachments/assets/bf57e6c9-2c8a-426a-a794-c69028432042">

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All versions

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Component: [Dynamic Page](https://sap.github.io/fundamental-ngx/#/core/dynamic-page)
The Pinned state for the icon ‘Pin header’ within all screens is not displayed with a contrast ratio of at least 3.0:1

#### Please provide stackblitz example(s).
You can refer ngx example, it is reproducible. (https://stackblitz.com/run?file=src%2Fapp%2Fdynamic-page-responsive-example.component.ts)

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes, issue is raised by an accessibility expert.

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**
